tasks/c++/easy/ReverseString.cpp

```diff
@@ -1,11 +1,17 @@
 #include <iostream>
 #include <string>
+using namespace std;
 
-std::string reverseString(const std::string &str) {
+string reverseString(const string &str) {
     // TODO: Implement the reverseString function
```

Comment:
You need to implement the actual reverseString function here, otherwise the code won't work. Other files look good so i can merge this PR after the reverseString function is added.